source/index.html

```diff
@@ -1,12 +1,248 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Пинк</title>
   </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
-
+  <body class="page__body">
+    <header class="page-header">
+      <a class="page-header__logo-name logo-name" href="index.html">
+        <img class="logo-name__header-image" src="img/logo-name-mobile-header.svg" width="74" height="23" alt="Мобильное приложение «Pink»" style="background-color: black;">
```

Comment:
Смотри, формально это не ошибка по БЭМ, но с точки зрения сути БЭМа получается ерунда.
У тебя должен быть блок независимый, а его элементы должны принадлежать только ему без оглядки на окружение.
А у тебя элемент называется `logo-name__header-image`, ты его привязываешь к хэдеру. Звучит как зависимость от местоположения.

Reply:
Не нужно инлайнить стили
```suggestion
        <img class="logo-name__header-image" src="img/logo-name-mobile-header.svg" width="74" height="23" alt="Мобильное приложение «Pink»">
```

Reply:
Просто без этого логотипа вообще не видно. Он же белый. 
Убрал

Reply:
Да и плевать, стили начнем делать и все будет видно сразу